kedro-datasets/pyproject.toml

```diff
@@ -220,8 +219,8 @@ test = [
     "scikit-learn>=1.0.2,<2",
     "scipy>=1.7.3",
     "packaging",
-    "SQLAlchemy~=1.2",
-    "tables~=3.8.0; platform_system == 'Windows'",  # Import issues with python 3.8 with pytables pinning to 3.8.0 fixes this https://github.com/PyTables/PyTables/issues/933#issuecomment-1555917593
+    "SQLAlchemy>=1.2",
+    "tables>=3.8.0; platform_system == 'Windows'",  # Import issues with python 3.8 with pytables pinning to 3.8.0 fixes this https://github.com/PyTables/PyTables/issues/933#issuecomment-1555917593
     "tables~=3.6; platform_system != 'Windows'",
```

Comment:
And actually, maybe a good moment to update this pin

Reply:
I checked the latest release is still 3.9.2 so I think this should still be getting the latest version